firstvoices/backend/tasks/utils.py

```diff
@@ -0,0 +1,2 @@
+ASYNC_TASK_START_TEMPLATE = "Task started. Additional info: %s."
```

Comment:
Just to double check-- does the formatted log line include the name of the task function?

Reply:
Yes, the celery logger provides the base useful info such as
1. Timestamp
2. Mainprocess/PoolWorker
3. Task name
4. Task Id

So I only kept the additional info in the log message.